metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/publish/PublishUtils.java

```diff
@@ -33,7 +34,12 @@ private PublishUtils() {
 
   public static PerfInfo createPerfInfo(MeasurementNode stageNode) {
     PerfInfo perfInfo = new PerfInfo();
-
+    MeasurementNode timeInterval = stageNode.findChild(StatisticsBucket.TIME_INTERVAL);
+    // latency buckets
+    if (timeInterval != null) {
+      perfInfo.initInterval(timeInterval);
```

Comment:
latency bucket is not belongs to perfInfo, right?

Reply:
Yes. I move it to the class **OperationPerf**, as I thought that every operation have it's own latency 
```java
public class OperationPerf {
  private String operation;

  private Map<String, PerfInfo> stages = new HashMap<>();

  private Map<String, BucketInfo> buckets = new HashMap<>();
```